editoast/src/modelsv2/work_schedules.rs

```diff
@@ -0,0 +1,42 @@
+use crate::schema::TrackRange;
+use chrono::NaiveDateTime;
+use chrono::Utc;
+use derivative::Derivative;
+use editoast_derive::ModelV2;
+use strum::FromRepr;
+
+use serde::Deserialize;
+use serde::Serialize;
+use utoipa::ToSchema;
+
+#[derive(Debug, Clone, Serialize, Deserialize, Derivative, ModelV2, ToSchema)]
+#[model(table = crate::tables::work_schedule_group)]
+#[derivative(Default(new = "true"))]
+pub struct WorkScheduleGroup {
+    pub id: i64,
+    #[derivative(Default(value = "Utc::now().naive_utc()"))] // Default does not seem to work ?
```

Comment:
Indeed `NaiveDateTime` doesn't implement the `Default` trait.

```suggestion
    #[derivative(Default(value = "Utc::now().naive_utc()"))]
```